frontend/components/vault/VaultLiquidationTransactionTable.vue

```diff
@@ -0,0 +1,158 @@
+                <FormatCurrency
+                    v-if="vaultTransaction.incentiveConstantDai"
+                    :value="vaultTransaction.incentiveConstantDai"
+                    currency="DAI"
+                />
+                <span v-else class="opacity-50">Unknown</span>
+            </div>
+        </div>
+        <div class="flex justify-between">
+            <div>Potential gross profit</div>
+            <div>
+                <template v-if="vaultTransaction.grossProfitDai">
+                    +<FormatCurrency :value="vaultTransaction.grossProfitDai" currency="DAI" />
+                </template>
+                <span v-else class="opacity-50">Unknown</span>
+            </div>
+        </div>
+        <div class="flex justify-between">
+            <div>
+                Transaction fee
+                <span v-if="vaultTransaction.transactionFeeLiquidationDai" class="opacity-50"
+                    >(~ <FormatCurrency :value="vaultTransaction.transactionFeeLiquidationEth" currency="ETH" />)</span
+                >
+            </div>
+            <div>
+                <template v-if="vaultTransaction.transactionFeeLiquidationDai">
+                    -<FormatCurrency :value="vaultTransaction.transactionFeeLiquidationDai" currency="DAI" />
+                </template>
+                <span v-else class="opacity-50">Unknown</span>
+            </div>
+        </div>
+        <div class="flex justify-between font-bold">
+            <div>Potential net profit</div>
+            <div>
+                <FormatCurrency
+                    v-if="vaultTransaction.netProfitDai"
+                    :value="vaultTransaction.netProfitDai"
+                    currency="DAI"
+                />
+                <span v-else class="opacity-50">Unknown</span>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import { VaultTransactionNotLiquidated } from 'auctions-core/src/types';
+import { formatToAutomaticDecimalPoints } from 'auctions-core/src/helpers/formatToAutomaticDecimalPoints';
+import TimeTill from '~/components/common/formatters/TimeTill.vue';
+import FormatCurrency from '~/components/common/formatters/FormatCurrency.vue';
+import Explain from '~/components/common/other/Explain.vue';
+import AnimatedArrow, { ArrowDirections } from '~/components/common/other/AnimatedArrow.vue';
+
+export default Vue.extend({
+    components: {
+        TimeTill,
+        FormatCurrency,
+        Explain,
+        AnimatedArrow,
+    },
+    props: {
+        vaultTransaction: {
+            type: Object as Vue.PropType<VaultTransactionNotLiquidated>,
+            required: true,
+        },
+    },
+    computed: {
+        incentiveRelativePercentage(): String {
+            return (
+                formatToAutomaticDecimalPoints(
+                    this.vaultTransaction.incentiveRelativeDai.div(this.vaultTransaction.debtDai).times(100)
+                ) + '%'
```

Comment:
You shouldn't both compute and format things in one function. Instead, I suggest to compute the value here and then use `FormatPercentage` or other format _component_ to format it (also because the component will support animation in comparison to the `formatToAutomaticDecimalPoints`)